README.md

```diff
@@ -1 +1,38 @@
-# easy-ui
+# Easy UI
+
+> Resources, components, and design guidelines for shaping the shipper experience at EasyPost.
```

Comment:
does `shaping the shipper experience` need to have more of a UI slant to it or is it clear from `Easy UI` above that this repo deals with the UI at EasyPost?

Reply:
i think im ok with shaping the shipper experience tbh. though, if we wanted to express easy ui as the design system, we could say its the design system for creating consistent (shipper) experiences at easypost; although i dont have a suggestion off the top of my head to form that into a cohesive sentence lol